### ISSUE TITLE
Packer is grumpy about repo name.

### ISSUE THREAD

**Issue by curiositycasualty** [YaleUniversity/packer-provisioner-goss#65]
```
$ packer init ./config.pkr.hcl
Error: Invalid plugin type

  on ./config.pkr.hcl line 3, in packer:
   3:         goss = {
   4:             version = ">= 3.1.4"
   5:             source = "github.com/YaleUniversity/packer-provisioner-goss"
   6:         }

Plugin source "github.com/yaleuniversity/packer-provisioner-goss" has a type
with the prefix "packer-", which isn't allowed because it would be redundant to
name a Packer plugin with that prefix. If you are the author of this plugin,
rename it to not include the prefix.

$ packer --version
1.8.5
```

**Comment by F21** [comment]
This would also be resolved once #58 is resolved.

**Comment by curiositycasualty** [comment]
Closing own stale issues.
